skimage/restoration/_denoise.py

```diff
@@ -429,6 +429,10 @@ def _wavelet_threshold(img, wavelet, threshold=None, sigma=None, mode='soft',
     """
     wavelet = pywt.Wavelet(wavelet)
 
+    # original_extent is used to workaround PyWavelets issue #80
+    # odd-sized input results in output with 1 extra sample after recon
```

Comment:
will change recon->reconstruction  (i.e. reconstruction of the image from the wavelet coefficients).  An alternative term would be synthesis.